engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html

```diff
@@ -0,0 +1,18 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    User Access - Console
+{/content-title}
+{#content-title-section1}
+    User Access platform group Notification
```

Comment:
Is `Notification` necessary? It was removed from other templates.
```suggestion
    User Access platform group
```

Reply:
Seems useless indeed